json-rpc/src/methods.rs

```diff
@@ -80,6 +80,72 @@ async fn get_metadata(service: JsonRpcService, _: Vec<Value>) -> Result<BlockMet
     Ok(BlockMetadata { version, timestamp })
 }
 
+/// Returns transactions by range
+async fn get_transactions(
+    service: JsonRpcService,
+    params: Vec<Value>,
+) -> Result<Vec<TransactionView>> {
+    let p_start_version: Number = serde_json::from_value(params[0].clone())?;
+    let p_limit: Number = serde_json::from_value(params[1].clone())?;
```

Comment:
nit: 
you can just do `let start: u64 = serde_json::from_value(params[1].clone())?;`
Take a look into get_events
no need to import Number and introduce p_* variables